diff --git a/node_modules/axios/index.d.ts b/node_modules/axios/index.d.ts
index abc4d93..2b1576e 100644
--- a/node_modules/axios/index.d.ts
+++ b/node_modules/axios/index.d.ts
@@ -309,6 +309,12 @@ export interface AxiosRequestConfig<D = any> {
   transitional?: TransitionalOptions;
   signal?: GenericAbortSignal;
   insecureHTTPParser?: boolean;
+   /**
+  * 接口是否登录鉴权（配置此参数后，会携带jwtToken）
+  */
+  needAuth?: boolean;
+  urlValue?: Record<string, string | number>;
+  hideErrorMessage?: boolean;
   env?: {
     FormData?: new (...args: any[]) => object;
   };
